torch/nn/modules/loss.py

```diff
@@ -778,7 +778,8 @@ class SmoothL1Loss(_Loss):
     :math:`x` and :math:`y` arbitrary shapes with a total of :math:`n` elements each
     the sum operation still operates over all the elements, and divides by :math:`n`.
 
-    beta is an optional parameter that defaults to 1.
+    beta is an optional parameter that defaults to 1. When beta is set to 0, we call out directly
+    to l1_loss. Passing a negative value in for beta will result in an exception.
```

Comment:
nit: you don't need to include implementation details here (call out directly to l1_loss).  You can just say something like, "Note: when beta is set to 0, this is equivalent to `:module:L1Loss` (not sure the formatting).

Reply:
(I think it's :class: judging from some of the other code)